pkg/udev/mockdata.go

```diff
@@ -83,6 +84,7 @@ func MockDiskDetails() (MockOsDiskDetails, error) {
 	diskDetails.Wwn = device.GetPropertyValue(UDEV_WWN)
 	diskDetails.Uid = device.GetUid()
 	diskDetails.FileSystem = UDEV_FS_NONE
+	diskDetails.Mountpoint = "/" // always take the disk mounted at /
```

Comment:
Let the default option be empty ( `""`), as the default FileSystem is none.

Reply:
@kmova  Should I change it everywhere, we use Filesystem type? If no filesystem exists, the CR will have not have the field., But internally, in the `DiskInfo` we pass it as None.
Also, should the default mountpoint also be removed?

Reply:
If we have it set as "None", can we keep it as empty.. and then data structure set as omit if empty?

Reply:
@kmova  I have already set it as `omitempty`. But omitempty works only if the struct is a pointer right?. "None" is used only inside the code for easier comparisons. The value in CR will be empty.
But the struct will show it as `FileSystemInfo:{}`

Reply:
cool. got it. This function is about OS disk and assumption is made that it is mounted under `/`. Only used for unit tests.